pytorch_lightning/trainer/training_loop.py

```diff
@@ -483,6 +491,17 @@ def run_training_epoch(self):
             if early_stop_epoch or self.fast_dev_run:
                 break
 
+        # process epoch outputs
+        if isinstance(model, (LightningDistributedDataParallel, LightningDataParallel)):
+            model = model.module
+
+        if self.is_overriden('training_epoch_end', model=model):
+            epoch_output = model.training_epoch_end(outputs)
+            _, _, log_epoch_metrics, callback_epoch_metrics, _ = self.process_output(
+                epoch_output)
```

Comment:
```suggestion
            _, _, log_epoch_metrics, callback_epoch_metrics, _ = self.process_output(epoch_output)
```

Reply:
done! I wrote it a bit differently to avoid having a line break character `\` after `=`.